name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/editor/ClientInput.java

```diff
@@ -62,6 +62,49 @@
     // compatibility: the value used to be stored in the AbstractHistoricView
     private static final String REPORTING_PERIODS_KEY = "AbstractHistoricView"; //$NON-NLS-1$
 
+    public static final String DEFAULT_RELATIVE_BACKUP_FOLDER = "backups"; //$NON-NLS-1$
+
+    public enum BACKUP_MODE
```

Comment:
I propose to name the Enum with a normal Java name (BackupMode) and extract it from the ClientInput class (because it is used in other places as well). There is no need to have that the inner class of ClientInput.